glusterfs/server/setup.sls

```diff
@@ -58,14 +58,18 @@ glusterfs_vol_{{ name }}:
 {%- endif %}
 
 glusterfs_vol_{{ name }}_start:
+  {%- if force_compatibility %}
+  cmd.run:
+    - name: |
+        gluster volume info {{ name }} | grep "Status: Started" ||  gluster volume start {{ name }}
```

Comment:
Fixed and verified that unless works correctly and now correctly shows whether the volume was started by not showing as changed. Thanks for the quick review!